generator/internal/descriptor_utils_test.cc

```diff
@@ -467,6 +467,141 @@ INSTANTIATE_TEST_SUITE_P(
       return std::get<0>(info.param);
     });
 
+class CreateServiceNameToCommentMappingTest : public CreateServiceVarsTest {};
```

Comment:
`using CreateServiceNameToCommentMappingTest = CreateServiceVarsTest` should suffice here.  No need for inheritence I believe.

Reply:
Done for here and above.